loleaflet/src/control/Control.Header.js

```diff
@@ -585,26 +580,15 @@ L.Control.Header = L.Control.extend({
 	},
 
 	_setCanvasSizeImpl: function (container, canvas, property, value, isCorner) {
-		if (!value) {
+		if (!value)
 			value = parseInt(L.DomUtil.getStyle(container, property));
-		}
-		else {
-			L.DomUtil.setStyle(container, property, value + 'px');
-		}
 
 		var scale = this.canvasDPIScale();
-		if (property === 'width') {
-			canvas.width = Math.floor(value * scale);
-			if (!isCorner)
-				this._canvasWidth = value;
-			// console.log('Header._setCanvasSizeImpl: _canvasWidth' + this._canvasWidth);
-		}
-		else if (property === 'height') {
-			canvas.height = Math.floor(value * scale);
-			if (!isCorner)
-				this._canvasHeight = value;
-			// console.log('Header._setCanvasSizeImpl: _canvasHeight' + this._canvasHeight);
-		}
+		L.DomUtil.setStyle(container, property, value + 'px');
```

Comment:
Previously the setStyle was not called in the !value case, now it is always, it is intended?

Reply:
Hello, yes it was intended, this function almost never called without value.